requirements.txt

```diff
@@ -1,77 +1,180 @@
+icu==58.2
+idna==2.8
+imageio==2.5.0
+imagesize==1.1.0
+intel-openmp==2019.3
+ipykernel==5.1.0
+ipyparallel==6.2.3
+ipython==7.4.0
+ipython_genutils==0.2.0
+ipywidgets==7.4.2
+jedi==0.13.3
+jinja2==2.10.1
+jpeg==9b
+jsonschema==3.0.1
+jupyter==1.0.0
+jupyter_client==5.2.4
+jupyter_console==6.0.0
+jupyter_core==4.4.0
 kiwisolver==1.0.1
-matplotlib==2.2.3
-monty==1.0.3
-more-itertools==4.3.0
-mpmath==1.0.0
-natsort==5.3.3
-networkx==2.1
-numpy==1.15.0
-palettable==3.1.1
-pandas==0.23.4
-parso==0.3.1
-patsy==0.5.0
-pexpect==4.6.0
-pickleshare==0.7.4
-Pillow==5.2.0
-Pint==0.8.1             
-pluggy==0.7.1
-prompt-toolkit==1.0.15
+libedit==3.1.20181209
+libffi==3.2.1
+libgcc-ng==8.2.0
+libgfortran-ng==7.3.0
+libpng==1.6.37
+libsodium==1.0.16
+libstdcxx-ng==8.2.0
+libtiff==4.0.10
+libuuid==1.0.3
+libxcb==1.13
+libxml2==2.9.9
+llvmlite==0.28.0
+locket==0.2.0
+markupsafe==1.1.1
+matplotlib==3.0.3
+mistune==0.8.4
+mkl==2019.3
+mkl_fft==1.0.10
+mkl_random==1.0.2
+more-itertools==7.0.0
+mpc==1.1.0
+mpfr==4.0.1
+mpmath==1.1.0
+msgpack-python==0.6.1
+natsort==6.0.0
+nbconvert==5.4.1
+nbformat==4.4.0
+ncurses==6.1
+networkx==2.3
+notebook==5.7.8
+numba==0.43.1
+numexpr==2.6.9
+numpy==1.16.3
+numpy-base==1.16.3
+olefile==0.46
+openssl==1.1.1b
+packaging==19.0
+pandas==0.24.2
+pandoc==2.2.3.2
+pandocfilters==1.4.2
+parso==0.4.0
+partd==0.3.10
+patsy==0.5.1
+pcre==8.43
+pexpect==4.7.0
+pickleshare==0.7.5
+pillow==6.0.0
+pint==0.9
+pip==19.0.3
+pluggy==0.9.0
+prometheus_client==0.6.0
+prompt_toolkit==2.0.9
+psutil==5.6.1
 ptyprocess==0.6.0
-py==1.5.4
-pycodestyle==2.4.0
-PyDispatcher==2.0.5
-PyWavelets==0.5.2
-Pygments==2.2.0
-pyparsing==2.2.0
-pytest-cov==2.5.1
-pytest==3.8.0
-python-dateutil==2.7.3
-pytz==2018.5
-pyzmq==17.1.2
-requests>=2.20.0
-ruamel.yaml==0.15.57
-scikit-image==0.14.0
-scikit-learn==0.19.2
-scipy==1.1.0
-setuptools==40.0.0
-simplegeneric==0.8.1
-six==1.11.0
-sphinx==1.7.8
+py==1.8.0
+pycifrw==4.4.1
+pycparser==2.19
+pygments==2.3.1
+pyopenssl==19.0.0
+pyparsing==2.4.0
+pyqt==5.9.2
+pyrsistent==0.14.11
+pysocks==1.6.8
+pytest==4.4.1
+pytest-cov==2.6.1
+python==3.7.3
+python-dateutil==2.8.0
+pytz==2019.1
+pywavelets==1.0.3
+pyyaml==5.1
+pyzmq==18.0.0
+qt==5.9.7
+qtconsole==4.4.3
+readline==7.0
+requests==2.21.0
+scikit-image==0.15.0
+scikit-learn==0.20.3
+scipy==1.2.1
+send2trash==1.5.0
+setuptools==41.0.0
+sip==4.19.8
+six==1.12.0
+snowballstemmer==1.2.1
+sortedcontainers==2.1.0
+sparse==0.7.0
+sphinx==2.0.1
 sphinx_bootstrap_theme==0.6.4
+sphinxcontrib-applehelp==1.0.1
+sphinxcontrib-devhelp==1.0.1
+sphinxcontrib-htmlhelp==1.0.2
+sphinxcontrib-jsmath==1.0.1
+sphinxcontrib-qthelp==1.0.2
+sphinxcontrib-serializinghtml==1.1.3
+sqlite==3.28.0
 statsmodels==0.9.0
-sympy==1.2
-tabulate==0.8.2
+sympy==1.4
+tblib==1.3.2
+terminado==0.8.2
+testpath==0.4.2
+tk==8.6.8
 toolz==0.9.0
-tornado==5.1
-tqdm==4.24.0
+tornado==6.0.2
+tqdm==4.31.1
 traitlets==4.3.2
-traits==4.6.0
+traits==5.1.1
 transforms3d==0.3.1
-urllib3>=1.24.2
+urllib3==1.24.1
```

Comment:
why did this one get downgraded? I think I recently bumped this up to resolve a security warning.

Reply:
resolved by cde41c1